farmbot_ext/test/test_helper.exs

```diff
@@ -1,3 +1,4 @@
 Mox.defmock(MockPreloader, for: FarmbotExt.API.Preloader)
+Mox.defmock(MockConnectionWorker, for: FarmbotExt.AMQP.ConnectionWorker.Network)
```

Comment:
I think we may want to come up with a better naming scheme for mocked modules. 
maybe something like `Mox.defmock(FarmbotExt.AMQP.ConnectionWorker.Mock, for: FarmbotExt.AMQP.ConnectionWorker.Network)`
or something